codegen/src/test/java/org/web3j/codegen/unit/gen/Setup.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.codegen.unit.gen;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.testcontainers.shaded.com.google.common.io.Files;
+
+import static java.io.File.separator;
+
+public class Setup {
+    static File temp = Files.createTempDir();
+    static File classAsFile =
+            new File(
+                    String.join(
+                            separator,
+                            temp.getAbsolutePath(),
+                            "test",
+                            "src",
+                            "integrationTest",
+                            "java",
+                            "org",
+                            "com",
+                            "generated",
+                            "contracts",
+                            "GreeterTest.java"));
+
+    @BeforeAll
+    public static void setUp() throws IOException, ClassNotFoundException {
+        File greeterJavaWrapper =
+                new File(
+                        "/home/alexander/Documents/dev/web3j/web3j/web3j/codegen/src/test/resources/java");
```

Comment:
We shouldn't have a file defined with an absolute path with your home directory in it.

Reply:
fixed